setup.py

```diff
@@ -8,20 +8,28 @@
 from setuptools import setup, find_packages
 
 
-def get_version():
+def load_version():
     """Returns the version found in nilearn/version.py
 
     Importing nilearn is not an option because there may dependencies
     like nibabel which are not installed and setup.py is supposed to
     install them.
     """
-    locals_dict = {}
+    # load all vars into globals, otherwise
+    #   the later function call using global vars doesn't work.
     globals_dict = {}
     with open(os.path.join('nilearn', 'version.py')) as fp:
-        exec(fp.read(), globals_dict, locals_dict)
+        exec(fp.read(), globals_dict)
 
-    return locals_dict['__version__']
+    return globals_dict
 
+
+def is_installing():
+    # Allow command-lines such as "python setup.py build install"
+    return len(sys.argv) > 1 and 'install' in sys.argv
```

Comment:
setuptools is recommended by the [Python Packaging User Guide](https://packaging.python.org/en/latest/current.html) over distutils.

It was used before I changed setup.py for example back in [2012](https://github.com/nilearn/nilearn/blob/2d2c50159164159482f5047b36a3f6f4e31a3aa2/setup.py)

For completeness it is also listed in the dependencies in README.rst.